src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/ProblemDetails.cs

```diff
@@ -18,33 +17,28 @@ public class ProblemDetails
         /// (e.g., using HTML [W3C.REC-html5-20141028]).  When this member is not present, its value is assumed to be
         /// "about:blank".
         /// </summary>
-        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "type")]
         public string Type { get; set; }
 
         /// <summary>
         /// A short, human-readable summary of the problem type.It SHOULD NOT change from occurrence to occurrence
         /// of the problem, except for purposes of localization(e.g., using proactive content negotiation;
         /// see[RFC7231], Section 3.4).
         /// </summary>
-        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "title")]
         public string Title { get; set; }
 
         /// <summary>
         /// The HTTP status code([RFC7231], Section 6) generated by the origin server for this occurrence of the problem.
         /// </summary>
-        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "status")]
         public int? Status { get; set; }
 
         /// <summary>
         /// A human-readable explanation specific to this occurrence of the problem.
         /// </summary>
-        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "detail")]
         public string Detail { get; set; }
 
         /// <summary>
         /// A URI reference that identifies the specific occurrence of the problem.It may or may not yield further information if dereferenced.
         /// </summary>
-        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "instance")]
         public string Instance { get; set; }
```

Comment:
We could annotate these with `DataMember`, but we still need to do something custom with `Extensions`. In addition, we need to figure out how to get swagger document generators to play nice with it. Most generators are aware of `JsonExtensionDataAttribute` and produce the right json schema when they encounter it

Reply:
So what happens here, does this just stop being nice?

Reply:
Should we implement a custom formatter for this? A custom API Description provider?

Reply:
People care about what the output looks like, we can't regress this.

Reply:
> Should we implement a custom formatter for this? A custom API Description provider?

Need to figure that out. Most swagger document generators look at attributes on the `Type`, and we don't have a knob to say this property on this type is an extension data. We could make another attribute and have swagger generators know about it. I thought we could do this in a follow up

Reply:
Tracking follow up via https://github.com/aspnet/AspNetCore/issues/6202